package.json

```diff
@@ -11,13 +11,24 @@
     "start": "cross-env NODE_ENV=development tsc-watch --onSuccess \"node .\"",
     "prebuild": "npm-run-all clean lint",
     "build": "tsc",
+    "build:no-check": "tsc",
```

Comment:
What does :no-check do to build btw?

Reply:
it just skips prebuild, so no clean no lint. actually now you bring it up we shouldn't skip clean